app/_includes/post.html

```diff
@@ -0,0 +1,28 @@
+<section id="post">
+
+  <article class="post row">
+
+    <div class="page-header">
```

Comment:
I think we should keep the header here. Just for the aesthetics. I know it's usable but without the header is even worse.